pkg/performanceprofile/controller/performanceprofile/components/kubeletconfig/kubeletconfig.go

```diff
@@ -98,6 +99,18 @@ func New(profile *performancev2.PerformanceProfile, profileMCPLabels map[string]
 		kubeletConfig.ReservedSystemCPUs = string(*profile.Spec.CPU.Reserved)
 	}
 
+	if opts.MixedCPUsEnabled {
+		sharedCPUs, err := cpuset.Parse(string(*profile.Spec.CPU.Shared))
+		if err != nil {
+			return nil, err
+		}
+		reservedCPUs, err := cpuset.Parse(string(*profile.Spec.CPU.Reserved))
+		if err != nil {
+			return nil, err
+		}
+		kubeletConfig.ReservedSystemCPUs = reservedCPUs.Union(sharedCPUs).String()
+	}
+
```

Comment:
Just wondering if we should be doing some sort validation to ensure that the CPUs are non-overlapping for `shared` and `reserved` fields?

Reply:
Hey @swatisehgal Thank you for taking the time to review the PR.
We do perform validation during the profile application using admission hooks.
see:
https://github.com/openshift/cluster-node-tuning-operator/pull/853/commits/a36f6f027eb2750b14ab2a111cd4ce3c8b2e6c20#diff-1a001c69ad51c074896e9bc0c708d605c59f65bd4fe82dd95a6f882358ac82f8R160